src/Particular.PlatformSample/AppLauncher.cs

```diff
@@ -55,19 +56,8 @@ public void Monitoring(int port, string logPath, string transportPath)
             platformJob.AddProcess(proc);
         }
 
-        public void ServicePulse(int port, int serviceControlPort, int monitoringPort, string defaultRoute)
+        public void ServicePulse(int port)
         {
-            var config = GetResource("Particular.configs.app.constants.js");
-
-            config = config.Replace("{Version}", typeof(ServicePulse).Assembly.GetCustomAttribute<ServicePulseVersionAttribute>().Version);
-            config = config.Replace("{DefaultRoute}", defaultRoute ?? "/dashboard");
-            config = config.Replace("{ServiceControlPort}", serviceControlPort.ToString());
-            config = config.Replace("{MonitoringPort}", monitoringPort.ToString());
-
-            var configPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"platform\servicepulse\js\app.constants.js");
```

Comment:
The `app.constants.js` file itself should also be removed from the project.

Reply:
There are likely even more changes that need to be made, for example see the stuff in the project file.

Reply:
@bording What are the other changes that you had in mind?